apph-front/src/privateRoutes.tsx

```diff
@@ -0,0 +1,12 @@
+import { Route, Routes } from 'react-router-dom';
+import { UserProfileContainer } from './static/containers/UserProfileContainer';
+import { SignIn } from './static/components/SignIn';
+
+export const PrivateRoutes = (): JSX.Element => {
+  return (
+    <Routes>
+      <Route path="/login" element={<SignIn />} />
+      <Route path="/me" element={<UserProfileContainer />} />
```

Comment:
Je te conseil d'ajouter des `exact` quand tu veux une url précise 
```suggestion
      <Route exact path="/login" element={<SignIn />} />
      <Route exact path="/me" element={<UserProfileContainer />} />
```

Reply:
y a plus besoin de 'exact' dans la version v6 du react route